invenio_rdm_records/vocabularies/utils.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+# Copyright (C) 2020 Northwestern University.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Utils for vocabulary code."""
```

Comment:
moderate: if it is only used for vocabularies, why couldn't it be part of said class? or at least file? totally biased comment since im not a fan of *util* files/modules. **Does not block merge**

Reply:
Agreed to move it to Vocabulary.py